core/src/main/java/org/fao/geonet/kernel/search/LuceneQueryBuilder.java

```diff
@@ -960,21 +960,16 @@ private void addBBoxQuery(Map<String, Set<String>> searchCriteria, BooleanQuery
     private void addPrivilegeQuery(LuceneQueryInput luceneQueryInput, BooleanQuery query) {
         // Set user groups privileges
         Set<String> groups = luceneQueryInput.getGroups();
-        String editable$ = luceneQueryInput.getEditable();
-        boolean editable = BooleanUtils.toBoolean(editable$);
+        Set<String> editableGroups = luceneQueryInput.getEditableGroups();
```

Comment:
@Delawen Apologies as could not check this, I see now it's merged, but I see you added `groupEdit` field that is used in `luceneQueryInput.getEditableGroups`, but I'm missing where this field is filled? 

To be honest, not very clear where the previous `editable` parameter used in `luceneQueryInput.getEditable()` was send.